tests/testthat/test_azimuth.R

```diff
@@ -0,0 +1,31 @@
+# Tests for azimuth
+set.seed(42)
+
+library(Seurat)
+library(SeuratData)
+library(Azimuth)
+
+
+reference <- LoadData("pbmcref", "azimuth")
+query <- LoadData("ifnb")
```

Comment:
You can't assume that these datasets are pre-installed since they underlying packages are not listed as dependencies in the package's DESCRIPTION (simply relying on `SeuratData` is not sufficient). 

Aside from that, these datasets are too large to use for testing. Individual tests should be quick so that the entire suit can run in less than five minutes. 

The first thing to do is decide how to generate a small dataset for testing. From there, we can chat about the right way to include it in the package.